src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class = "calendar calendar--first-day--sun calendar--month-length--31">
+      <li class = "calendar__day"></li>
+      <li class = "calendar__day"></li>
```

Comment:
The space before and after `=` sign is redundant, fix it
```suggestion
      <li class="calendar__day"></li>
```